### DISCUSSION TITLE
dble支持单库分表吗？

### DISCUSSION THREAD

**Discussion by ican2056** [actiontech/dble#2829] [Q&A] [answered]
您好，dble支持单库分表吗，就是同一个库里面同样的表存储为table_1，table_2，table_3... 这样需要怎样配置？

**Comment by PanternBao** [comment, marked as answer]
dble不支持单库分表，至于不支持的原因：
1. 需要实现二级路由计算 
2. 需要下发时候做表名转换
3. 没有比原生分区表更好的收益
4. 未来不利于扩容

可以使用单实例分库达到等价的效果scheme1.table，scheme2.table，scheme3.table...
此外，完全可以用mysql原生分区表取代单库分表
